badges/badges.md

```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
 
-### What are badges?
-The badges are a complement system to the Ladder - while ladder outlines career paths, the badges reward behaviors and achievements we respect and appreciate in our fellows.
+### What are the badges?
+The badges are a complementing tool to the Ladder - while ladder outlines career paths, badges reward behaviors and achievements we respect and appreciate in our team members.
 
 ### How can I create a badge?
-Simple. Badges must meet the following criteria:
-#### 1. A badge achievement must be a measurable, objective, pass fail criteria.
-###### Example Badge: *Good presenter*
-Do: Presented to an audience of at least 50 people
-Don't: Had a nice, well structured presentation
+Simply. Badges must meet the following criteria:
+#### 1. Badge achievement must be a measurable, objective, and pass-fail criteria.
+###### Example: *Good speaker*
+Surely: Prepared an educative talk for an audience of ~50+ people
+Not really: Uploaded a few well-structured presentations
 
-#### 2. A badge must be achievable by one person in reasonable time
-###### Example Badge: *CR Knight*
-Do: Performed 1000 code reviews
-Don't: Invested 100000 hours reviewing code
+#### 2. A badge must be achievable by one person in a reasonable time
+###### Example: *Code Review Knight*
+Surely: Had objective insights that helped many people get on the right track
+Not really: Invested 1000 hours in reviewing code
 
-#### 3. Badges must reflect behavior that are contributing to the company as well as to the team of the badge winner
-###### Example badge: *CR Knight*
-Do: Performed 1000 code reviews
-Don't: Performed a code review in full plate armor [^1]
+#### 3. Badges must reflect behavior that are contributing to the team, the community and the company
+###### Example: *Mentor Mage*
+Surely: Helped his team-mates understand the codebase so that they became more independent at work
```

Comment:
Switching the badge was a good idea - try to come up with a badge that's measurable, objective, and have clear pass-fail criteria.   
"Helped his teammates understand the codebase" is unclear, and would require a proxy for it

Reply:
fixed